src/dockerfile.ml

```
@@ -211,6 +219,7 @@ let cmd fmt = ksprintf (fun b -> [ `Cmd (`Shell b) ]) fmt
 let cmd_exec cmds : t = [ `Cmd (`Exec cmds) ]
 let expose_port p : t = [ `Expose [ p ] ]
 let expose_ports p : t = [ `Expose p ]
+let arg a ?default () : t = [ `Arg (a, default) ]
```

Why not:
```suggestion
let arg ?default a : t = [ `Arg (a, default) ]
```
This way, you don't have to include the `unit` at the end.

---

As it's a label argument, not sure it's an issue :)